### ISSUE TITLE
Prevent asset flipping

### ISSUE THREAD

**Issue by RuneStone0** [bitshares/bsips#259]
Dear BitShares community,

I've noticed that some users are attempting to try their luck with "asset flipping" (similar to domain flipping) - or at least it seems like it. Take for example: https://wallet.bitshares.org/#/account/blokzinciri.org who registered multiple "core assets" such as https://wallet.bitshares.org/#/asset/WEALTH but never use it for anything.

In my opinion this can become a problem for BitShares because someone could acquire a lot of "core assets" and wait for someone to buy it for a high premium. This could potentially scare organizations away from creating new gateways if they cannot get the "core assets" they are looking for.

Obviously, its first-come-first-serve just like when you buy a traditional web domain such as .net .com etc. but I think BitShares could benefit from preventing excessive "core asset" registration by for example by:
1) increase the costs of creating new "core assets"
2) add yearly fee to hold a "core asset"
3) require a certain volume of transactions between the "core asset" and (BTS, bitUSD...) to hold rights to a core asset

These are just examples. Please let me know what you think...?

**Comment by jmjatlanta** [comment]
My $0.02: Adding / increasing fees will do more to scare people away than paying a high price to acquire a "vanity" coin name.

**Comment by RuneStone0** [comment]
@jmjatlanta I tend to agree, those were just suggestions. Personally, I'm leaning more towards a yearly fee to hold "core assets" and/or require a certain volume to prevent assets just being bought up and parked without any real use.
